RELEASE.md

```diff
@@ -55,6 +55,7 @@
 * `kedro.io.DataCatalogWithDefault` is being deprecated, to be removed entirely in 0.18.0.
 
 ## Thanks for supporting contributions
+[Punit Vara](https://github.com/punitvara),
```

Comment:
```suggestion
```
0.17.6 was the last version we did this for - now we just just rely on the [automatic GitHub feature](https://github.com/kedro-org/kedro/graphs/contributors) to track contributors 🙂